source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to synchronize data from a replica set to a 
+sharded cluster using :program:`mongosync`. 
+
+.. note::
+
+   ``mongosync`` doesn't support sync from a sharded cluster to a replica set.
+
+To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
```

Comment:
Consider moving this to the Learn More section

```suggestion
To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
```

Reply:
Good idea, I also don't think this sentence needs to be on this page when it is already on the topology landing page.